### PR TITLE
Tweak : Image Box widget

### PR DESCRIPTION
**Pull request by rodolphebertozzo**
- Added a caption option for the image caption
- Added possibility of box shadow for the image 

fix : https://github.com/elementor/elementor/issues/2131

Screenshots :
légende style
légende
ombre de boite

### CODE REVIEW

**Comment by rodolphebertozzo** [comment]
@matipojo

**Comment by matipojo** [comment]
Hi, @rodolphebertozzo Thanks!
Can you please attach some screenshots with how it looks like on the Editor Panel/ Fronend?

**Comment by rodolphebertozzo** [comment]
@matipojo : 
matipojo

**Comment by rodolphebertozzo** [comment]
Hi @matipojo, can you help me, I can't get the caption to appear, yet the method seems to be the right one, do you have an idea?

File: includes/widgets/image-box.php [hunk 1]
```diff
@@ -579,6 +676,12 @@ protected function render() {
 			$html .= '<figure class="elementor-image-box-img">' . $image_html . '</figure>';
 		}
 
+		$image_caption = $this->get_image_caption( $attachment );
```
Comment (matipojo) [inline includes/widgets/image-box.php:679]: Where is the initialization of the `$attachment`

**Review by matipojo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**